consensus/clique/src/main/java/org/hyperledger/besu/consensus/clique/blockcreation/CliqueMiningCoordinator.java

```diff
@@ -33,6 +39,20 @@ public CliqueMiningCoordinator(
     this.miningTracker = miningTracker;
   }
 
+  @Override
+  public void onResumeMining() {
+    if (miningTracker.isSigner(blockchain.getChainHeadHeader())) {
+      LOG.info("Resuming block production operations");
```

Comment:
nit: Other thing I was thinking - is this really an "info" level log - or can it be reduced to debug? Seems like something we wouldn't need to see all the time.

Reply:
IMO it can be interesting to see that the block production start and stop without changing the log level to debug. Do you really think that we need to change it ?

Reply:
No - just wanted to ask.
I feel the logging Besu is a generally bit verbose/noisy - so I tend to be pretty hard with what should be at info/debug/trace.
Really dropping out of sync is the problem - the fact that mining is paused is _kinda_ an obvious side effect - so maybe "Lost Sync" is in info, and stopping mining is debug.
But seriously, happy to let it lie and push this in.